### PR TITLE
SWARM-1808 override properties with system env

### PR DESCRIPTION
**Pull request by johnAls**
Make it possible to override values in projects-*yml with environment variables.
Its need if running in docker container and you need to push variabels from exempel ansible

- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

**Comment by kenfinnigan** [comment]
ok to test

**Comment by kenfinnigan** [comment]
@bobmcwhirter, could you take a look at this as to me it seems fine, but you've dealt with the yaml stuff a lot more than me!

File: core/container/src/test/resources/withenvvalues.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+foo:
+  on:
+    startup: "${env.BOOL}"
+    somestring: "${env.URL:http://someurl}"
```
Comment (heiko-braun) [inline core/container/src/test/resources/withenvvalues.yml:4]: neat!

**Comment by heiko-braun** [comment]
looks good to me

**Review by bobmcwhirter — APPROVED** [review]

**Pull request merged**